features/controller_specs/cookies.feature

```diff
@@ -0,0 +1,36 @@
+Feature: Cookies
+
+  There is different ways to test cookies in controller's tests be we recommend
+  to use `cookies` as below.
+
+  You can use strings or symbols to fetch or set your cookies because `cookies`
+  support indifferent access.
```

Comment:
Swap out '`cookies` support' for 'the `cookies' method supports'

Reply:
Thanks. Done with https://github.com/rspec/rspec-rails/pull/2127/commits/54104d8ea3ce6c4f75be17518f34593799930371